Makefile

```diff
@@ -0,0 +1,32 @@
+VIRTUAL_ENV ?= .venv
+
+.PHONY: run
+run: $(VIRTUAL_ENV)
+	$(VIRTUAL_ENV)/bin/pip install -r requirements.txt
+	npm install
```

Comment:
can we make this `npm install || yarn install` ?

Reply:
feel free to change (what exactly does || do in shell?)

Reply:
logical, lazy evaluated OR, if with `cmd1 || cmd2` cmd1 fails return code != 0, cmd2 will run